### PR TITLE
update to the consensus archive page

### LINKED ISSUE #938: Updating the Consensus Vote Archive page
I am looking to update the page with the V6 fixlnseqlocks vote information, while I am here I think the page may benefit from the following changes:

1) Show the newest vote on top vs the oldest it shows currently
2) Adding links to dcrdata where blocks are mentioned.
3) adding links to DCP where missing.
4) Showing vote start, end and activated in a table format
5) Adding abstain %, previously only had Yes and No percentages

Can also add quorum numbers and the total number of votes for each but wanted to keep the amount of information to a minimum as we have the information at https://voting.decred.org/ as well as https://explorer.dcrdata.org/agenda. 

example of format changes:

Screenshot from 2019-05-10 00-55-18

Screenshot from 2019-05-10 01-04-54

### PR DESCRIPTION
**Pull request by zubairzia0**
Updating page with the fixlnseqlocks vote and changed the flow of the page from the newest vote to oldest. Added links to dcrdata and abstain %.
(Closes #938)

### CODE REVIEW

**Comment by s-ben** [comment]
Ran locally, LGTM. Listing most recent votes first makes more sense. The links to the new vote charts in dcrdata seem useful.

File: docs/governance/consensus-rule-voting/consensus-vote-archive.md [hunk 1]
```diff
@@ -4,71 +4,98 @@
...
+
+#### Voting Results
+
+|Choice|Percentage|
+|------|----------|
+|Abstain|22.58%|
+|No|1.07%|
+|Yes|76.34%|
+
+---
```
Comment (jholdstock) [inline docs/governance/consensus-rule-voting/consensus-vote-archive.md]: Please remove this line break from the end of the page

File: docs/governance/consensus-rule-voting/consensus-vote-archive.md [hunk 2]
```diff
@@ -4,71 +4,98 @@
...
 
-**Agenda ID:**  sdiffalgorithm
 
-Change stake difficulty algorithm as defined in DCP0001
+**Modify sequence lock handling as defined in [DCP0004](https://github.com/decred/dcps/blob/master/dcp-0004/dcp-0004.mediawiki)**
 
-Specifies a proposed replacement algorithm for determining the stake difficulty (commonly called the ticket price). This proposal resolves all issues with a new algorithm that adheres to the referenced ideals.
+In order to fully support the Lightning Network, the current sequence lock consensus rules need to be modified.
+
+|**Voting Start:**|**Voting End:**|**Activated:**|
```
Comment (jholdstock) [inline docs/governance/consensus-rule-voting/consensus-vote-archive.md]: Column headers do not need to be bold, and do not need trailing commas

File: docs/governance/consensus-rule-voting/consensus-vote-archive.md [hunk 3]
```diff
@@ -4,71 +4,98 @@
...
-## V4
+### Update Sequence Lock Rules
 
-### Change PoS Staking Algorithm
+Agenda ID: [fixlnseqlocks](https://explorer.dcrdata.org/agenda/fixlnseqlocks)
 
-**Agenda ID:**  sdiffalgorithm
 
-Change stake difficulty algorithm as defined in DCP0001
+**Modify sequence lock handling as defined in [DCP0004](https://github.com/decred/dcps/blob/master/dcp-0004/dcp-0004.mediawiki)**
```
Comment (jholdstock) [inline docs/governance/consensus-rule-voting/consensus-vote-archive.md]: Please format DCP names with a hyphen (eg. `DCP0004` should become `DCP-0004`)

File: docs/governance/consensus-rule-voting/consensus-vote-archive.md [hunk 4]
```diff
@@ -4,71 +4,98 @@
...
 
-Specifies a proposed replacement algorithm for determining the stake difficulty (commonly called the ticket price). This proposal resolves all issues with a new algorithm that adheres to the referenced ideals.
+In order to fully support the Lightning Network, the current sequence lock consensus rules need to be modified.
+
+|**Voting Start:**|**Voting End:**|**Activated:**|
+|-----------------|---------------|--------------|
+[326656](https://explorer.dcrdata.org/block/326656)|[334720](https://explorer.dcrdata.org/block/334720)|[342784](https://explorer.dcrdata.org/block/342784)|
```
Comment (jholdstock) [inline docs/governance/consensus-rule-voting/consensus-vote-archive.md]: Split block numbers with commas (eg. `326656` should be come `326,656`)

**Comment by zubairzia0** [comment]
Let me know how those look.

**Comment by jholdstock** [comment]
Thats great, thanks

**Pull request merged**